packages/libs/client/src/composePactCommand/utils/addKeyset.ts

```diff
@@ -0,0 +1,12 @@
+import { addData, IAddKeyset } from './addData';
+
+/**
+ * Helper to add keyset to the data property for {@link IPactCommand.payload}
+ *
+ * @public
+ */
+export const addKeyset: IAddKeyset = (
+  name: string,
+  pred: string,
```

Comment:
Nitpick: not common to shorten `pred`

Reply:
We used this to keep familiarity with a [Keyset definition](https://pact-language.readthedocs.io/en/stable/pact-reference.html#keyset-definition-1)